CHANGELOG.md

```diff
@@ -14,6 +14,13 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
 
 - Removed faulty `within_the_last()` method from `sdk.queries.alerts.filters.alert_filter.DateObserved`.
 
+### Changed
+- Updated Alert `resolve-alert` and `reopen-alert` endpoints to `update-state` endpoint.
```

Comment:
I don't think we need to add this to the Changelog since it happens behind the scenes